utils/run_tests.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# DOCS
+# This script is adopted from st2-run-pack-tests, but has been changed
+# to allow for faster iteration, provided that the user has already prepared
+# an environment in which the tests can run.
+
+##################
+# Common functions
+##################
+
+function join { local IFS="$1"; shift; echo "$*"; }
+
+####################
+# Script beings here
+####################
+
+PACK_PATH=/opt/stackstorm/packs/arteria-packs
+ST2_REPO_PATH="./st2/"
+
+source venv/bin/activate
+
+SCRIPT_PATH=$(readlink -f $0)
+DIRECTORY_PATH=$(dirname ${SCRIPT_PATH})
+
+PACK_NAME=$(basename ${PACK_PATH})
+PACK_TESTS_PATH="${PACK_PATH}/tests/"
+
+SENSORS_PATH="${PACK_PATH}/sensors/"
+ACTIONS_PATH="${PACK_PATH}/actions/"
+ETC_PATH="${PACK_PATH}/etc/"
+
+# Bail early if no tests are found, this way we don't need to wait for
+# environment set up.
+if [  ! -d ${PACK_TESTS_PATH} ]; then
+    echo "Running tests for pack: ${PACK_NAME}"
+    echo "No tests found."
+    exit 0
```

Comment:
Perhaps you should do an exit != 0 here, as this will pass otherwise as can be seen by the latest travis run.